ipyleaflet/here.py

```diff
@@ -99,11 +102,28 @@ def basemap_to_tiles(**kwargs):
      'attribution': '&copy; <a href="https://here.com">HERE.com</a>',
      'name': 'HERE'}
     """
-    params = dict(
+    return TileLayer(
         url = build_tiles_url(**kwargs),
         min_zoom = kwargs.get("min_zoom", 1),
         max_zoom = kwargs.get("max_zoom", 18),
         attribution = '&copy; <a href="https://here.com">HERE.com</a>',
         name = "HERE",
+        **kwargs
     )
-    return params  # better use TileLayer?
+
+
+basemaps = Bunch(
```

Comment:
Comment: I wonder if we should not change this name to something like `heremaps`? And do something equivalent for future providers in this directory?

Reply:
But then it would be a different name to memorize for each provider.

Reply:
Sure, I agree